src/package_test.ts

```diff
@@ -7,4 +7,13 @@ describe('package', () => {
         const v2 = require('../package-lock.json').version;
         expect(v1).to.equal(v2);
     });
+
+    it('package-lock should only reference npm', () => {
+        const deps = require('../package-lock.json').dependencies;
+        for (const key in deps) {
+            const dep = deps[key];
+            const resolved = new URL(dep.resolved);
+            expect(resolved.hostname).to.equal('registry.npmjs.org');
```

Comment:
Can we also add a HTTPS check here too while we at it. 

```suggestion
            expect(resolved.hostname).to.equal('registry.npmjs.org');
            expect(resolved.protocol ).to.equal('https);
```

Its too easy to mistakenly switch to the HTTP registry and I doubt either you or I would notice it.